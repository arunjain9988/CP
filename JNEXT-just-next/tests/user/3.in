8
5
1 5 4 8 3
10
1 4 7 4 5 8 4 1 2 6
7
2 7 9 9 8 7 3
3
3 1 2
3
1 3 2
4
2 9 9 2
3
1 2 3
5
1 1 1 1 1