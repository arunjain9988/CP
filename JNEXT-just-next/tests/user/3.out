15834
1474584162
2837799
321
213
9229
132
-1